bot/cogs/modlog.py

```diff
@@ -14,8 +14,7 @@
 from discord.abc import GuildChannel
 from discord.ext.commands import Bot
 
-from bot.constants import Channels, Colours, Emojis, Event, Icons, Keys, Roles, URLs
-from bot.constants import Guild as GuildConstant
+from bot.constants import Channels, Colours, Emojis, Event, Guild as GuildConstant, Icons, Keys, Roles, URLs
```

Comment:
This line scares me terribly and now I need someone to hug, can we not split it over multiple lines?
```suggestion
from bot.constants import (
    Channels, Colours, Emojis,
    Event, Guild as Guild Constant, Icons,
    Keys, Roles, URLs
)
```